Dockerfile

```diff
@@ -12,9 +11,17 @@ RUN apk add --update bash \
   && cd $GOPATH/src/${REPO_PATH} \
   && go get -t ./... \
   && go test ./... \
-  && go build \
+  && LDFLAGS="$(${GOPATH}/src/github.com/Financial-Times/service-status-go/buildinfo/ldFlags.sh)" \
+  && go build -ldflags="${LDFLAGS}" \
   && mv content-preview /content-preview-app \
   && apk del go git bzr \
   && rm -rf $GOPATH /var/cache/apk/*
```

Comment:
In Java services we typically rm the source after running Maven, so you might want to remove `.git` afterwards as well.